templates/taxi/driver_create.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>Create a Driver profile:</h1>
+  <form action="?next={% url 'login' %}" method="post">

```

Comment:
The `form` action attribute includes `?next={% url 'login' %}`. Ensure that this is intentional and correctly redirects after form submission. If not needed, consider removing it.